.taskcluster.yml

```diff
@@ -233,14 +233,17 @@ tasks:
             command:
               - "/bin/bash"
               - "-cx"
-              - "git clone --recursive --quiet ${repository} &&
+              - "apt-get update &&
+                 apt-get install -y musl &&
+                 rustup target add x86_64-unknown-linux-musl &&
+                 git clone --recursive --quiet ${repository} &&
                  cd rust-code-analysis &&
                  git -c advice.detachedHead=false checkout --recurse-submodules ${head_rev} &&
-                 cargo build --all --release &&
-                 cargo package --all-features &&
-                 pushd rust-code-analysis-cli && cargo package --all-features && popd &&
-                 pushd rust-code-analysis-web && cargo package --all-features && popd &&
-                 cd target/release &&
+                 env CC=\"gcc\" CXX=\"g++\" cargo build --workspace --release --target x86_64-unknown-linux-musl &&
```

Comment:
Why do we need to define CC and CXX?

Reply:
To compile `tree-sitter` submodules, otherwise the `musl` target fails

Reply:
Why does it fail? I mean, why doesn't it automatically use the right compiler? What compiler does it try to use? Do you have an example of a failure?